examples/python/objectron/main.py

```diff
@@ -116,9 +116,9 @@ def log_ar_frames(samples: Iterable[SampleARFrame], seq: Sequence) -> None:
         rr.set_time_seconds("time", sample.timestamp)
         frame_times.append(sample.timestamp)
 
-        rr.log("world/camera", rr.ImageEncoded(path=sample.image_path))
         log_camera(sample.frame.camera)
         log_point_cloud(sample.frame.raw_feature_points)
+        rr.log("world/camera", rr.ImageEncoded(path=sample.image_path))
```

Comment:
Nit: a comment on the importance of the order could be useful here as an explanation for users as well.

Reply:
I'm so ashamed of it though...